frontend/src/scenes/dashboard/DashboardItem.tsx

```diff
@@ -326,12 +326,10 @@ export function DashboardItem({
                                     trigger={['click']}
                                     overlay={
                                         <Menu data-attr={'dashboard-item-' + index + '-dropdown-menu'}>
-                                            <Menu.Item
-                                                data-attr={'dashboard-item-' + index + '-dropdown-view'}
-                                                icon={<Icon />}
-                                                onClick={() => router.actions.push(link)}
-                                            >
-                                                {viewText}
+                                            <Menu.Item data-attr={'dashboard-item-' + index + '-dropdown-view'}>
```

Comment:
Q: Does the anchor tag take up 100% of the menu item? Otherwise it could lead to missed clicks.

Reply:
It does, tested this.

<img width="183" alt="" src="https://user-images.githubusercontent.com/5864173/128055989-a838012e-6c40-47a1-8114-48cb7f4911eb.png">